bedrock/firefox/templates/firefox/channel/base.html

```diff
@@ -46,18 +46,20 @@ <h2 class="c-sub-navigation-title">
 
   {% block channels %}{% endblock %}
 
-  {% call picto_card(
-    title=ftl('firefox-channel-see-something-that-isnt-working'),
+  <aside class="mzp-t-picto-center">
+  {% call picto(
+    image_url='img/icons/warning.svg',
     heading_level=2,
-    class='bugzilla',
-    base_el='section',
-    custom_desc=True) %}
-
-  <ul>
-    <li><a class="mzp-c-cta-link" href="https://bugzilla.mozilla.org/enter_bug.cgi">{{ ftl('firefox-channel-file-a-bug-now') }}</a></li>
-    <li><a class="mzp-c-cta-link" href="https://developer.mozilla.org/docs/Mozilla/QA/Bug_writing_guidelines">{{ ftl('firefox-channel-tips-for-filing-a-bug') }}</a></li>
-  </ul>
-  {% endcall %}
+    image_width=44,
+    title=ftl('firefox-channel-see-something-that-isnt-working'),
+    body=True,
+    ) %}
+      <ul>
+        <li><a class="mzp-c-cta-link" href="https://bugzilla.mozilla.org/enter_bug.cgi">{{ ftl('firefox-channel-file-a-bug-now') }}</a></li>
+        <li><a class="mzp-c-cta-link" href="https://support.mozilla.org/en-US/kb/contributors-guide-writing-good-bug">{{ ftl('firefox-channel-tips-for-filing-a-bug') }}</a></li>
```

Comment:
If we omit `/en-US` from this URL, SUMO will automatically redirect to a browser's preferred language.